test/localstack.ts

```diff
@@ -30,16 +37,18 @@ export const deployStack = async (app: App, stack: Stack) => {
     httpOptions: clientConfig as any,
   });
 
-  // @ts-ignore
-  sdkProvider.sdkOptions = {
+  if (clientConfig) {
     // @ts-ignore
-    ...sdkProvider.sdkOptions,
-    endpoint: clientConfig.endpoint,
-    s3ForcePathStyle: clientConfig.s3ForcePathStyle,
-    accessKeyId: clientConfig.credentials.accessKeyId,
-    secretAccessKey: clientConfig.credentials.secretAccessKey,
-    credentials: clientConfig.credentials,
-  };
+    sdkProvider.sdkOptions = {
+      // @ts-ignore
```

Comment:
?

Reply:
this was existing code that was moved. 

Might be assigning some not so public props :)

![image](https://user-images.githubusercontent.com/289213/187964743-50f68ec2-ab9b-4f1b-8f2a-107d29507842.png)